controller.py

```diff
@@ -609,7 +610,7 @@ def base_url(full_url):
             )
         except ValueError as e:
             log.error("Error in IntegrationClient.register", exc_info=e)
-            return INVALID_CREDENTIALS.detailed(repr(e))
+            return INVALID_CREDENTIALS.detailed(_(repr(e)))
```

Comment:
This one I don't think is even worth localizing -- we don't know what the string is to localize.